javascript/app.js

```diff
@@ -4,15 +4,18 @@ FinancialFreedom.config(['$routeProvider', '$locationProvider', function($routeP
 
     $routeProvider.when('/income', {
         templateUrl: 'partials/income_input.html',
-        controller: 'IncomeInputController'
+        controller: 'IncomeInputController',
+        controllerAs: 'incomectrl'
     })
     .when('/assets', {
         templateUrl: 'partials/assets_input.html',
-        controller: 'AssetsInputController'
+        controller: 'AssetsInputController',
+        controllerAs: 'assetsctrl'

```

Comment:
not a big deal, but for clarity it might be better if this (and others like it) were in camel case or snake case (probably snake case)